mender/templates/api-gateway/configmap.yaml

```diff
@@ -506,6 +506,8 @@ data:
             browserXssFilter: true
             customRequestHeaders:
               "X-Forwarded-Proto": "{{ $scheme }}"
+              "X-Mender-Version": "{{ .Chart.Version }}"
```

Comment:
Great stuff! In Alvaldi the headers are lowercase, but I guess it's fine

Reply:
yep, was just aligning with the header above - it _shouldn't_ really matter at all:  https://datatracker.ietf.org/doc/html/rfc7230#section-3.2 but if we come across anything I'll lowercase right away 😄